src/libYARP_math/include/yarp/math/RandScalar.h

```diff
@@ -65,13 +64,6 @@ class YARP_math_API yarp::math::RandScalar
      void init (int seed);
 
     /**
-    * Get the seed.
-    * @return the seed.
-    */
-     int getSeed ()
-     { return seed; }
-
-    /**
     * Generate a random number from a 
```

Comment:
This is breaking the API. Function should be removed only after they have been deprecated for some time, see https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md#deprecated-features .

Reply:
In general, have a way to return the original seed seems an interesting feature. What is the rationale behind the removal?

Reply:
I saw now the commit message. I think the proper way to correct this problem is to save the correct value of `seed` in the`init()` method: https://github.com/damn1/yarp/blob/7fc0a3fed6e8c0c700d891e6dfce9456330eea24/src/libYARP_math/src/RandScalar.cpp#L59 .

Reply:
Yes I talked with @drdanz about this and I saw `getSeed()` was never used but effectively removing a function withoute even deprecating it may be too drastic so I will do what you suggest, thanks

Reply:
@damn1 I might be missing something relevant, but how can you be sure that `getSeed()` is never used outside Yarp?

Keeping it doesn't hurt...